### ISSUE TITLE
Rule for all Programs

### ISSUE THREAD

**Issue by engineer-plus-plus** [objective-see/LuLu#593]
Is there a way to create a rule that blocks any program from connection to a certain host/port?  If not, I'd like to make such feature request.

**Comment by mdjunior** [comment]
In this issue, you can check the LuLu rule processing flow: https://github.com/objective-see/LuLu/issues/540.

But if you are thinking about blocking based on destination, you might want to use a blocklist for that. I believe that much of LuLu's processing is based on the source program, so the blocklist would be a way to block a destination.

**Comment by objective-see** [comment]
Yes, just specify `*` as the program path, in the "Add Rule" window ...and then specify the host/port for this "global rule"
![image](https://github.com/objective-see/LuLu/assets/11327812/c5cb93ce-6ba8-4446-9015-0f501a9e4871)

**Comment by cameronbraid** [comment]
Could you add to this the ability to allow from a list of IP addresses ?
